app/models/paper.rb

```diff
@@ -5,10 +5,12 @@ class Paper < ApplicationRecord
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references, dependent: :destroy
+  belongs_to :publication
 
   accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
-  validates_associated :authors
+  accepts_nested_attributes_for :publication, reject_if: proc { |attributes| attributes['name'].blank? }
```

Comment:
You use `proc { |attributes| attributes['name'].blank? }` in two places. Should we think about maybe assigning the proc to a variable or just encapsulating it as a method?